cmd/minikube/cmd/version.go

```diff
@@ -43,13 +48,57 @@ var versionCmd = &cobra.Command{
 			"minikubeVersion": minikubeVersion,
 			"commit":          gitCommitID,
 		}
+
+		if listPackagesVersions {
```

Comment:
`if listPackagesVersions && !shortVersion`
we don't need all this if `--short` is set

Reply:
good call !